oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -211,7 +238,17 @@ protected GoogleCredentials() {
    * @param accessToken initial or temporary access token
    */
   public GoogleCredentials(AccessToken accessToken) {
+    this(accessToken, null);
+  }
+
+  /**
+   * Constructor with explicit access token and quota project.
+   *
+   * @param accessToken initial or temporary access token
+   */
+  public GoogleCredentials(AccessToken accessToken, @Nullable String quotaProjectId) {
```

Comment:
you can remove this now we have createWithQuotaProject

Reply:
Since you have to pass the value from every credential anyways, you better call the method. So you keep interface clean as it was.

Reply:
UserCredentials constructor is passed in quotaprojectId. So we need this constructor in base class.